src/components/scene/editor.js

```diff
@@ -0,0 +1,38 @@
+var registerComponent = require('../../core/component').registerComponent;
+
+module.exports.Component = registerComponent('editor', {
+  schema: { default: true },
+
+  init: function () {
+    this.onKeydown = this.onKeydown.bind(this);
+  },
+
+  play: function () {
+    window.addEventListener('keydown', this.onKeydown, false);
+  },
+
+  pause: function () {
+    window.removeEventListener('keydown', this.onKeydown);
+  },
+
+  onKeydown: function (evt) {
+    // Alt + Ctrl + e
+    var shortcutPressed = evt.keyCode === 69 && evt.ctrlKey && evt.altKey;
+    if (!this.data || this.editorInjected || !shortcutPressed) { return; }
+    this.injectEditor();
+  },
+
+  injectEditor: function () {
+    var script = this.script = document.createElement('script');
+    script.type = 'text/javascript';
+    script.async = true;
+    script.onload = function () { console.log('A-Frame Editor Loaded'); };

```

Comment:
shouldn't the Editor do its logging? (and perhaps spit out `A-Frame Editor Version: 0.1.0`?)